examples/workflow_by_code.ipynb

```diff
@@ -38,6 +38,8 @@
     "    # install qlib\n",
     "    ! pip install --upgrade numpy\n",
     "    ! pip install pyqlib\n",
+    "    # The Google colab environment is a little out-of-date. We have to downgrade the pyyaml to be compatible with other packages",
```

Comment:
" # The Google colab environment is a little outdated. We have to downgrade the pyyaml to make it compatible with other packages",

Reply:
@Singh212dab Thanks so much.  It is fixed now.